code/index.html

```diff
@@ -1,30 +1,135 @@
-      <div class="answer">
-        <p>Lorem ipsum...</p>
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="style.css">
+  <link href="https://fonts.googleapis.com/css?family=New+Rocker|Roboto&display=swap" rel="stylesheet">
+  <title>Drummer Bootcamp</title>
+
+</head>
+
+<body>
+
+  <header>
+    <div class="video-container">
+      <video autoplay muted loop playsinline>
+        <source class="video-mobile" src="./videos/drummerVideo360.mp4" type="video/mp4"
+          alt="by Taryn Elliott from Pexels">
+        <source class="video-tablet" src="./videos/drummerVideo960.mp4" type="video/mp4"
+          alt="by Taryn Elliott from Pexels">
+        <source class="video-desktop" src="./videos/videoDrummer-1280.mp4" type="video/mp4"
+          alt="by Taryn Elliott from Pexels">
+        Your browser does not support the video tag.
+      </video>
+      <div class="overlay-header">
+        <h1 class="h1-header">Drummer Bootcamp</h1>
+        <h2 class="h2-header">Become an animal with our help!</h2>
       </div>
+
+    </div>
+  </header>
+
+  <section class="lead-paragraph">
+    <p>Become a drummer in just one week. We will learn you all the basic rock n' roll beats you need to know to start
+      your
+      own band. Grab a pair of sticks and join us for a ride!</p>
+  </section>
+
+  <Section class="signup-info">
+    <div class="form-container">
+      <form action="http://httpbin.org/anything" method="post">
+        <h2>Drummer Bootcamp</h2>
+        <h3>Sign Up</h3>
+        <label>
+          First Name
+          <input type="text" name="first-name" required>
+        </label>
+
+        <label>
+          Last Name
+          <input type="text" name="last-name" required>
+        </label>
+
+        <label>
+          E-mail
+          <input type="email" name="email" placeholder="name@mail.com" required>
+        </label>
+
+        <label>
+          Desired Bootcamp Week
+          <select name="week" id="week" required>
+            <option value="22">w 22</option>
+            <option value="23">w 23</option>
+            <option value="26">w 26</option>
+          </select>
+        </label>
+
+        <div class="instrument">
+          <label>
+            Do you play another instrument?
+            <input type="radio" name="instrument" value="yes">Yes
+            <input type="radio" name="instrument" value="no">No
+          </label>
+        </div>
+
+        <label>
+          If yes, what instrument?
+          <input type="text" name="first-name">
+        </label>
+
+        <label class="terms">
+          <input type="checkbox" name="conditions" value="yes">
+          I accecpt the terms & conditions<br>
+        </label>
+        <button type="submit">Rock on!</button>
+      </form>
+    </div>
+
+    <section class="accordion-container">
+      <section class="accordion">
+        <h1>FAQ</h1>
```

Comment:
Try to use only one `<h1>` per page. The `<h1>` should be the page heading, so maybe this would be suited as a `<h2>` heading instead?